website/docs/docs/dbt-cloud-apis/service-tokens.md

```diff
@@ -18,9 +18,9 @@ You can use service account tokens for system-level integrations that do not run
 
 You can assign as many permission sets as needed to one token. For more on permissions sets, see "[Enterprise Permissions](/docs/cloud/manage-access/enterprise-permissions)."
 
-## Generating service account tokens
+## Generate service account tokens
 
-To make a service token in dbt Cloud, follow these steps:
+You can generate service tokens if you have a Developer [license](/docs/cloud/manage-access/seats-and-users) and account admin [permissions](#permissions-for-service-account-tokens). To make a service token in dbt Cloud, follow these steps:
```

Comment:
@nghi-ly , i would double check with tania and matt about 'acct admin' link, i don't think they meant acct admin from the service token permission because you wouldn't have account admin initially if you're generating a service token. i think they meant account admin group ([rbac group](https://docs.getdbt.com/docs/cloud/manage-access/enterprise-permissions#account-roles)). i added the user/license group link just incase! 

```suggestion
You can generate service tokens if you have a Developer [license](/docs/cloud/manage-access/seats-and-users) and account admin [permissions](/docs/cloud/manage-access/about-user-access#permission-sets). To create a service token in dbt Cloud, follow these steps:
```

Reply:
good catch!